README.rst

```diff
@@ -224,6 +224,21 @@ debian
 
     An ephemeral container with which to run sysadmin tasks on the cluster
 
+get_db_name
+~~~~~~~~~~~
+
```

Comment:
Maybe add that a `container_name` variable is now required either in `tasks.py` or in the `invoke.yaml` file.